script.js

```diff
@@ -77,3 +77,107 @@ function myFunction(x) {
 
 const x = window.matchMedia('(max-width: 768px)');
 myFunction(x); // Call listener function at run time
+
+// modal work start
+const showItemInfo = {
+  title: document.getElementById('h-title').innerText,
+  image: document.getElementById('post-img').getAttribute('src'),
+  description: document.getElementById('desc').innerText,
+  technologies: {
+    tech1: document.getElementById('post-tech1').innerText,
+    tech2: document.getElementById('post-tech2').innerText,
+    tech3: document.getElementById('post-tech3').innerText,
+    tech4: document.getElementById('post-tech4').innerText,
+  },
+  liveVersion: 'https://samiullah997.github.io/Portfolio-Mobile/',
+  liveSource: 'https://github.com/samiullah997/Portfolio-Mobile',
+};
+const showModal = document.getElementById('show-modal');
+const showModal1 = document.getElementById('show-modal-1');
+const showModal2 = document.getElementById('show-modal-2');
+const showModal3 = document.getElementById('show-modal-3');
+const showModal4 = document.getElementById('show-modal-4');
+const showModal5 = document.getElementById('show-modal-5');
+const showModal6 = document.getElementById('show-modal-6');
+const closeModal = document.getElementById('close-modal');
+showModal.addEventListener('click', () => {
+  document.getElementById('modal').classList.add('active');
+  document.getElementById('postTitle').innerText = showItemInfo.title;
+  document.getElementById('tech1').innerHTML = showItemInfo.technologies.tech1;
```

Comment:
- [ ] you should put the project's information in an array and loop in the array  and use the information that you stored in order to create the cards  that will make things easy when you want to add a new card project to  your array (portfolio)